src/Stage/Parse/Parser.elm

```diff
@@ -835,6 +1065,68 @@ newlines =
     P.chompWhile ((==) '\n')
 
 
+checkIndent : (Int -> Int -> Bool) -> ParseProblem -> Parser_ ()
+checkIndent check error =
```

Comment:
Please let's leave some explanatory doc comment here. What are the two integers inside the `check` function (`intend` and `col`)? When does the parser fail and succeed? I feel like some examples of usage might be good for understanding too.

This is coming from somebody who didn't use `elm/parser`'s `indent` functionality yet (yeah...) - so that perhaps clears up why I want this documentation. Links to `elm/parser`'s documentation about `indent` and `col` are welcome!